src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -141,45 +118,55 @@ function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, u
             >
                 <MenuItem
                     title={keyTitle}
-                    icon={item.icon}
+                    icon={item.icon ?? ''}
                     iconType={CONST.ICON_TYPE_WORKSPACE}
                     onPress={item.action}
                     iconStyles={item.iconStyles}
                     iconFill={item.iconFill}
                     shouldShowRightIcon
-                    badgeText={getWalletBalance(isPaymentItem)}
+                    badgeText={getWalletBalance(false)}
```

Comment:
Why was this changed?

Reply:
Because the ```const isPaymentItem = item.translationKey === 'common.wallet';``` in this case is always false